Resources/doc/reference/configuration/match.rst

```diff
@@ -128,8 +128,11 @@ regular expressions.
 
 **type**: ``array``
 
-A list of additional HTTP status codes of the response for which to also apply
-the rule.
+By defaut, a rule will only match cacheable status codes: 200, 203, 300, 301,
```

Comment:
In that RFC, the list is longer: ‘e.g., 200, 203, 204, 206, 300, 301, 404, 405, 410, 414, and 501 in this specification’.